pyserini/search/__main__.py

```diff
@@ -1,33 +1,55 @@
 import re
```

Comment:
when you get a chance, please add in our standard boilerplate headers. and reranker below.